src/fdtransfer/fdtransferServer.cpp

```diff
@@ -267,7 +273,7 @@ bool FdTransferServer::sendFd(int fd, struct fd_response *resp, size_t resp_size
     return true;
 }
 
-static int single_pid_server(int pid) {
+static int single_pid_server(int pid, const char *path) {
```

Comment:
`single_pid_server` previously switched to the target's `net` namespace.
Now it depends on the path type, whether to switch or not.

Reply:
Correct. I'll add a check here